ticdc/ticdc-debezium.md

```diff
@@ -172,4 +784,18 @@ TiCDC Debezium 消息中的数据格式映射基本遵循 [Debezium 的数据类
 
 - 对于 String-likes 的数据类型，包括 Varchar、String、VarString、TinyBlob、MediumBlob、BLOB、LongBlob 等，当该列具有 BINARY 标志时，TiCDC 会将其按照 Base64 编码后以 String 类型表示；当该列没有 BINARY 标志时，TiCDC 则直接将其编码为 String 类型。而原生的 Debezium Connector 会根据 `binary.handling.mode` 以不同的编码方式进行编码。
 
-- 对于 Decimal 数据类型，包括 `DECIMAL` 和 `NUMERIC`，TiCDC 均会使用 float64 类型来表示。而原生的 Debezium Connector 会根据数据类型的不同精度采用 float32 或者 float64 的方式进行编码。
+- 对于 Decimal 数据类型，包括 DECIMAL 和 NUMERIC，TiCDC 均会使用 float64 类型来表示。而原生的 Debezium Connector 会根据数据类型的不同精度采用 float32 或者 float64 的方式进行编码。
+
+- TiCDC 将 REAL 转换为 DOUBLE；当长度为 1 时，将 BOOLEAN 转换为 TINYINT(1)。
+
+- 在 TiCDC 中，BLOB、TEXT、GEOMETRY、JSON 列没有默认值。
+
+- Debezium 将 FLOAT 类型的 `"5.61"` 转换为 `"5.610000133514404"`，但 TiCDC 不会。
+
+- TiCDC 在处理 FLOAT 时打印了错误的 `flen` [tidb#57060](https://github.com/pingcap/tidb/issues/57060)。
+
+- 当列的排序规则为 `utf8_unicode_ci` 且字符集为 null 时，Debezium 将 `charsetName` 转换为 `"utf8mb4"`，但 TiCDC 不会。
+
+- Debezium 会对 ENUM 元素进行转义，但 TiCDC 不会。例如，Debezium 将 ENUM 元素 ('c', 'd', 'g,''h') 编码为 ('c','d','g,\'\'h')。
```

Comment:
@wk989898 Please update the example here. It does not display well.
<img width="958" alt="image" src="https://github.com/user-attachments/assets/0e4dae79-ad5e-4c99-91cb-7d27c6333227" />